addons/l10n_rs_edi/models/account_move.py

```diff
@@ -0,0 +1,157 @@
+
+    @api.depends('country_code', 'move_type')
+    def _compute_show_delivery_date(self):
+        # EXTENDS 'account'
+        super()._compute_show_delivery_date()
+        for move in self:
+            if move.country_code == 'RS':
+                move.show_delivery_date = move.is_sale_document()
+
+    @api.depends("country_code", "move_type")
+    def _compute_l10n_rs_edi_is_eligible(self):
+        for move in self:
+            move.l10n_rs_edi_is_eligible = move.country_code == 'RS' and move.is_sale_document()
+
+    def _post(self, soft=True):
+        posted = super()._post(soft)
+        posted.l10n_rs_edi_state = 'to_send'
+        return posted
+
+    @api.depends('l10n_rs_edi_state')
+    def _compute_show_reset_to_draft_button(self):
+        # EXTENDS 'account'
+        super()._compute_show_reset_to_draft_button()
+        for move in self:
+            move.show_reset_to_draft_button = move.l10n_rs_edi_state != 'sent' and move.show_reset_to_draft_button
+
+    def button_draft(self):
+        # EXTENDS 'account'
+        self.write(
+            {
+                "l10n_rs_edi_error": False,
+                "l10n_rs_edi_attachment_file": False,
+                "l10n_rs_edi_attachment_id": False,
+            }
+        )
+        return super().button_draft()
+
+    def _l10n_rs_edi_send(self, xml, sendToCir):
+        self.ensure_one()
+        params = {
+            'requestId': self.l10n_rs_edi_uuid,
+            'sendToCir': 'Yes' if sendToCir else 'No'
+        }
+        url = DEMO_EFAKTURA_URL if self.company_id.l10n_rs_edi_demo_env else EFAKTURA_URL
+        headers = {
+            'Content-Type': 'application/xml',
+            'ApiKey': self.company_id.l10n_rs_edi_api_key,
+        }
+        try:
+            response = requests.post(url=url, params=params, headers=headers, data=xml, timeout=30)
+        except (Timeout, ConnectionError, HTTPError) as exception:
+            self.l10n_rs_edi_state = 'sending_failed'
+            error_message = _("There was a problem with the connection with eFaktura: %s", repr(exception))
+            self.l10n_rs_edi_error = error_message
+            self.message_post(body=error_message)
+            return [self.l10n_rs_edi_error]
+        if not response.ok:
+            self.l10n_rs_edi_state = 'sending_failed'
+            message = _("Sending Failed with Code: %s \n%s", response.status_code, json.loads(response.content.decode('utf-8')))
+            self.l10n_rs_edi_error = message
+            return [self.l10n_rs_edi_error]
+        dict_response = json.loads(response.content.decode('utf-8'))
+        self.l10n_rs_edi_state = 'sent'
+        self.l10n_rs_edi_error = False
+        self.l10n_rs_edi_invoice_id = dict_response.get('InvoiceId')
+        self.l10n_rs_edi_purchase_invoice_id = dict_response.get('PurchaseInvoiceId')
+        self.l10n_rs_edi_sales_invoice_id = dict_response.get('SalesInvoiceId')
```

Comment:
write?
You must commit after that. Check if you can commit using `_can_commit()`